state/accountsDB.go

```diff
@@ -841,7 +840,6 @@ func (adb *AccountsDB) printTrieStorageStatistics() {
 func (adb *AccountsDB) Commit() ([]byte, error) {
 	adb.mutOp.Lock()
 	defer func() {
-		adb.printTrieStorageStatistics()
```

Comment:
Remove this from `CommitInEpoch()` also

Reply:
And don't forget to add it to the `func (adb *AccountsDB) commit() ([]byte, error)` which is called from here and from `CommitInEpoch()`

Reply:
done